mne/io/curry/curry.py

```diff
@@ -3,508 +3,222 @@
 # License: BSD-3-Clause
 # Copyright the MNE-Python contributors.
 
-import os.path as op
 import re
-from collections import namedtuple
 from datetime import datetime, timezone
 from pathlib import Path
 
+import curryreader
```

Comment:
This and the `pandas` import should be nested in the function(s) that need them using `_soft_import`